xbmc/addons/AddonDatabase.cpp

```diff
@@ -405,7 +405,7 @@ bool CAddonDatabase::FindByAddonId(const std::string& addonId, ADDON::VECADDONS&
       builder.SetDescription(m_pDS->fv(3).get_asString());
       DeserializeMetadata(m_pDS->fv(4).get_asString(), builder);
       builder.SetChangelog(m_pDS->fv(5).get_asString());
-      builder.SetOrigin(m_pDS->fv(6).get_asString());
+      builder.SetOrigin(m_pDS->fv(7).get_asString());
```

Comment:
I think this change slipped in from another change and it's unrelated to this PR.

Reply:
i think the value `6` is correct here, because the query returns 7 fields numbered 0-6 - see https://github.com/xbmc/xbmc/pull/18181#issuecomment-671161984

Reply:
This is actually in a temp commit so should be ignored.

Reply:
^ yup. please ignore.